host-interaction/process/inject/use-process-replacement.yml

```diff
@@ -8,6 +8,8 @@ rule:
     att&ck:
       - Defense Evasion::Process Injection::Process Hollowing [T1055.012]
       - Defense Evasion::Reflective Code Loading [T1620]
+    mbc:
+      - Defense Evasion::Process Injection::Process Hollowing [E1055.012]
```

Comment:
This ID doesn't show up in the STIX data.

Reply:
Removed incorrect mapping

Reply:
Does this need to be removed here?